src/main/java/core/basesyntax/Ball.java

```diff
@@ -1,16 +1,27 @@
 package core.basesyntax;
 
 public class Ball {
-    private String color;
-    private int number;
+    private String ballColor;
+    private int ballNumber;
```

Comment:
We can just call it `color` and `number` because we already know the context because of the class name.

Reply:
Rename fields, please